source/settings/command-line-options.txt

```diff
@@ -96,6 +98,31 @@ value in the :guilabel:`Settings` panel.
      - Enable :ref:`automatic updates <enable-automatic-updates>`. Use 
        ``--no-autoUpdates`` to disable automatic updates. 
 
+   * - .. option:: --browserCommandForOIDCAuth
+     - Specify a shell command to start the browser for OIDC authentication 
+       when connecting to the server or logging into your Atlas Cloud account.
```

Comment:
A lot of these duplicate the documentation from https://www.mongodb.com/docs/compass/current/settings/config-file/#settings – is it a good idea to duplicate these?

Reply:
Oh I see I think I had initially misunderstood, so sorry! We could create includes for these and have them be shared? If so, I can create a small follow up ticket to this :)